File: src/main/rst/examples/Chapter4/ruby/ImplicitWaitExample01.rb
require 'rubygems' # not required for ruby 1.9 or if you installed without gem
require 'selenium-webdriver'

driver = Selenium::WebDriver.for :firefox
driver.manage.timeouts.implicit_wait = 10 # seconds

driver.get "http://somedomain/url_that_delays_loading"
begin
element = driver.find_element(:id => "some-dynamic-element")
ensure
driver.quit
end

File: src/main/rst/examples/Chapter4/Javascript/ImplicitWaitExample01.js
var webdriver = require('selenium-webdriver');
var driver = new webdriver.Builder().forBrowser('firefox').build();
driver.manage().timeouts().implicitlyWait(10);
driver.get('http://somedomain/url_that_delays_loading');
var element = driver.findElement(webdriver.By.name('some-dynamic-element'));